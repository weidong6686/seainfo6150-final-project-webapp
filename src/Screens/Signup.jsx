import React from 'react';
import { Link } from 'react-router-dom';

const Signup = (props) => {

    // const redirect = props.location.search ? props.location.search.split("=")[1] : '/';

    return (<div className="form">
    <form>
      <ul className="form-container">
        <li>
          <h2>Create Account</h2>
        </li>
        <li>
          <label htmlFor="name">
            Name
          </label>
          <input type="name" name="name" id="name">
          </input>
        </li>
        <li>
          <label htmlFor="email">
            Email
          </label>
          <input type="email" name="email" id="email">
          </input>
        </li>
        <li>
          <label htmlFor="password">Password</label>
          <input type="password" id="password" name="password">
          </input>
        </li>
        <li>
          <label htmlFor="rePassword">Re-Enter Password</label>
          <input type="password" id="rePassword" name="rePassword">
          </input>
        </li>
        <li>
          <button type="submit" className="button primary">Register</button>
        </li>
        <li>
          Already have an account?
          <Link to="./Signin" className="button secondary text-center" >Sign In your Shopee Mart account</Link>
        </li>

      </ul>
    </form>
  </div>

    )


}
export default Signup;