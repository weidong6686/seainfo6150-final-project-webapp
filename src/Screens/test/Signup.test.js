import React from "react";
import Signup from "../Signup.jsx";

describe("Signup tests", () => {
  it("renders correctly", () => {
    const { container } = (<Signup />);
    expect(container).toMatchSnapshot();
  });
});