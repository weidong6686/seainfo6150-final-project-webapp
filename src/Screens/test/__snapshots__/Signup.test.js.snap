// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup tests renders correctly 1`] = `undefined`;
